packages/nextjs/package.json

```diff
@@ -86,8 +86,8 @@
   },
   "peerDependencies": {
     "next": "^13.5.4 || ^14.0.3",
-    "react": ">=18",
-    "react-dom": ">=18"
+    "react": ">=18 || 19.0.0-beta",
```

Comment:
given that `>=18` has no upper bound, is this change necessary? I'm guessing it might be because of the prerelease identifier.

In any case,  I think this should be:
```
>=18 || >=19.0.0-beta
```

Per https://semver.npmjs.com

Reply:
Oh damn it, I think I messed up here. 
It 100% needs to be `>=` or the check on the prerelease will always be false.
Let me test quick to make sure